Exercise_4/css/partials/side-menu/_small-screen.scss

```diff
@@ -0,0 +1,79 @@
+@import '../variables';
+
+/*SIDE-MENU*/
+
+@media only screen and (max-width: $small-screen_max-width) {
+
+  .side-menu {
+    cursor: pointer;
+    display: flex;
+    font-size: 2rem;
+    margin: 0 2rem;
+    width: 10vw;
+  }
+
+  .side-menu__content-wrapper {
+    display: none;
+    padding: 1rem;
+  }
+
+  .side-menu:focus .side-menu__content-wrapper {
```

Comment:
It should move into `.side-menu {`. So it will look like:
```
.side-menu {
  ..
  &:focus {
    .side-menu__content-wrapper {
    ...
  }
}
```